### PR TITLE
Skip some non-Windows tests on Windows

### PR DESCRIPTION
**Pull request by MinchinWeb**
Some tests will never pass on Windows due to differences in filesystems between
Windows and Linux.

### CODE REVIEW

File: pelican/tests/test_contents.py [hunk 1]
```diff
@@ -943,7 +943,7 @@ def test_not_save_as_draft(self):
             source_path=os.path.join('dir', 'foo.jpg'),
             context=self.settings.copy())
 
-        expected_save_as = os.path.join('dir', 'foo.jpg')
+        expected_save_as = os.path.join('dir', 'foo.jpg').replace('\\', '/')
```
Comment (avaris) [inline pelican/tests/test_contents.py]: see `pelican.utils.posixize_path`.

Comment (MinchinWeb) [reply pelican/tests/test_contents.py]: done!

**Comment by MinchinWeb** [comment]
bump?

**Comment by justinmayer** [comment]
Many thanks to @MinchinWeb for the enhancement and to @avaris for the review. 😁

**Comment by justinmayer** [comment]
This is included in the just-released Pelican v4.1.3, courtesy of [AutoPub](https://justinmayer.com/projects/autopub/).

**Pull request merged**